prdoc/pr_4301.prdoc

```diff
@@ -0,0 +1,11 @@
+title: New runtime api to check if a validator has pending pages of rewards for an era.
+
+doc:
+  - audience: Node Dev
```

Comment:
```suggestion
  - audience: Node Dev | Runtime Dev
```

I think it is worth pursuing a new audience type here, wdyt?

Reply:
Added Runtime user which is defined in schema as `Anyone using the runtime. This can be a token holder or a dev writing a front end for a chain.`